From b65d0a79a2030a2570f250a1739218b7b11a9f06 Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Thu, 23 Jul 2020 10:28:23 +0700
Subject: [PATCH] r8a7795: Makefile: support fixed device memory for PVR

We will enable feature "Fixed memory area" for PVR to use static memory
address instead of using CMA of kernel.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 build/linux/r8a7795_linux/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/build/linux/r8a7795_linux/Makefile b/build/linux/r8a7795_linux/Makefile
index 977a4df..cd4f6d0 100644
--- a/build/linux/r8a7795_linux/Makefile
+++ b/build/linux/r8a7795_linux/Makefile
@@ -81,10 +81,10 @@ KERNEL_COMPONENTS := srvkm $(DISPLAY_CONTROLLER)
 
 SYS_CFLAGS := -march=armv8-a
 
-SUPPORT_FIXED_DEVICE_MEMORY ?= 0
+SUPPORT_FIXED_DEVICE_MEMORY ?= 1
 ifeq ($(SUPPORT_FIXED_DEVICE_MEMORY),1)
-SUPPORT_FIXED_DEVICE_MEMORY_ADDR := 0x0700000000ULL
-SUPPORT_FIXED_DEVICE_MEMORY_SIZE := 0x40000000ULL
+SUPPORT_FIXED_DEVICE_MEMORY_ADDR := 0x0610000000ULL
+SUPPORT_FIXED_DEVICE_MEMORY_SIZE := 0x20000000ULL
 endif
 
 # Yocto 64bit SDK settings
-- 
2.17.1

